lib/fetch-libc++.sh

```diff
@@ -19,14 +19,21 @@ MIRRORS=(\
   "https://alpha.mirror.svc.schuermann.io/files/tock"\
 )
 
+if test -x /usr/bin/shasum; then
+  CHECK_SHA_CMD="shasum -a 256 -c"
+else
+  CHECK_SHA_CMD="sha256sum -c"
+fi
+
 let FOUND=0
 
 # Try from each mirror until we successfully download a .zip file.
 for MIRROR in ${MIRRORS[@]}; do
   URL=$MIRROR/$ZIP_FILE
   echo "Fetching libc++ from ${MIRROR}..."
   echo "  Fetching ${URL}..."
-  wget -O $ZIP_FILE  "$URL" && (echo "$GCC_SHA $ZIP_FILE" | sha256sum -c)
+  # Note: There must be two space characters for `shasum` (sha256sum doesn't care)
```

Comment:
Two space characters where? what is this referring to?

Reply:
Between the correct sha and the file name:

```
❯ echo "1e6f33cd5f44acefebc1ef677db4be33829a5228  README.md" | shasum -a 256 -c
README.md: OK
❯ echo "1e6f33cd5f44acefebc1ef677db4be33829a5228 README.md" | shasum -a 256 -c
shasum: standard input: no properly formatted SHA checksum lines found
```